libnvme-rs/src/error.rs

```diff
@@ -0,0 +1,65 @@
+use snafu::Snafu;
```

Comment:
Is this same as the [errors](https://github.com/openebs/mayastor/blob/develop/nvmeadm/src/error.rs) in nvmeadm

Reply:
Yes, with the addition of `LookupHostError` and `AddCtrlrError`.